packages/terra-application-docs/src/terra-dev-site/tool/terra-dev-site/UpgradeGuides.f/v7.0.0.s.tool.mdx

```diff
@@ -0,0 +1,150 @@
+      new TerraDevSite({
+        <Put your config here>
+      }),
+  ],
+}
+```
+
+Full config options can be found on the [config page](/dev_tools/cerner-terra-application-docs/terra-dev-site/configuration).
+
+To see how the old configuration maps to the new configuration, see the table below:
+
+|Old Config Value|New Config Value|Description|
+|---|---|---|
+|`navConfig.navigation.index`|**removed**|The site index page is now assumed to be the first `primaryNavigationItem` in the list.|
+|`navConfig.navigation.primaryNavigationItems`|`primaryNavigationItems`| PrimaryNavigationItems are no longer nested under the navigation object.|
+|`navConfig.navigation.primaryNavigationItems.path`|`primaryNavigationItems.path`|unchanged.|
+|`navConfig.navigation.primaryNavigationItems.text`|`primaryNavigationItems.label`|Changed to label to align with terra-application api naming.|
+|`navConfig.navigation.primaryNavigationItems.pageType`|`primaryNavigationItems.contentExtension`|Pages no longer support being generated from multiple page types. No sites ever used the functionality. PageType has been changed to contentExtension to eliminate confusion with terra-application pages.|
+|`navConfig.navigation.primaryNavigationItems.capabilities`|**removed**|This configuration has been removed. All pages except for home will have access to devTools|
+|`pagesConfig`|**removed**|`pagesConfig` allowed consumers to provide their own page config. It was poorly documented, complex and no one ever did.|
+|`generatePages`|`additionalSearchDirectories`|Instead of filling out this complex object. Terra dev site searches the 'terra-dev-site' folder by default for content with the specified content Extensions. Consumers that wish to search other directories now can provided fully qualified paths which will also be searched|
+|`hotReloading`|**removed**|True in dev false in prod|
+|`npmPackage`|**removed**|Only the root npm package is used.|
+|`sideEffectImports`|`sideEffectImportFilePaths`|The name now includes 'filePaths' to better describe the expected input.|
+|`placeholderSrc`|**removed**|No longer used.|
+|`readMeContent`|`primaryNavigationItems.additionalContent`|No longer a one off, consumers can now add additional content to each primary navigation item that would not normally be found by searching.|
+|`appConfig.title`|`titleConfig.title`|Moved to a titleConfig object to align with terra-application-navigation config.|
+|`appConfig.favicon`|`faviconFilePath`|No longer nested under appConfig. The name now includes 'filePaths' to better describe the expected input.|
+|`appConfig.themes`|**removed**|The available themes are now pulled from the terra-theme.config file via the webpack global variable `TERRA_THEME_CONFIG`.|
+|`appConfig.defaultTheme`|`defaultTheme`|No longer nested under appConfig.|
+|`appConfig.locales`|**removed**|Deprecated as of terra-dev-site 6.23, now removed.|
+|`appConfig.defaultLocale`|`defaultLocale`|No longer nested under appConfig.|
```

Comment:
above comment, is there a default used?

Reply:
Defaults to en, This is documented on the config page, but not here because it hasn't changed from v6 to v7